thorlcr/msort/tsorts.cpp

```diff
@@ -262,6 +288,11 @@ class CWriteIntercept : public CSimpleInterface
         if (!dataFileIO)
         {
             dataFileIO.setown(dataFile->open(IFOread));
+            if (compressedOverflowFile)
+            {
+                dataFileIO.setown(createCompressedFileReader(dataFileIO));
```

Comment:
node:  This will buffered in the compressed file code and the serial stream.  No simple fix/solution I suspect to avoid the duplication - so will merge as is.